src/Lykke.Job.Messages/QueueConsumers/SmsQueueConsumer.cs

```diff
@@ -82,10 +84,14 @@ private void InitQueues()
                 nameof(HandleSmsRequestAsync),
                 $"SMS: Phone confirmation. Receiver: {request.PhoneNumber}, UTC: {DateTime.UtcNow}");
 
-            var msgText = await _smsTextGenerator.GenerateConfirmSmsText(request.MessageData.ConfirmationCode);
+            var msgText = await _templateFormatter.FormatAsync(nameof(SmsConfirmationTemplate), request.PartnerId, "EN",

```

Comment:
Wouldn't it be better to use const or enum instead of hard coded string?

Reply:
https://lykkex.atlassian.net/browse/LWDEV-4615